### PR TITLE
Style sharing code page

### LINKED ISSUE #32: Style the "sharing code" page
## Purpose
Similar to #27, the sharing code page is very barebones and needs to be styled:
image

## Proposed Solution
Page should use Bulma for responsivity and follow the same style as the other pages.

## Acceptance Criteria
Please list the requirements the implementation of this feature should meet.
- [ ] Page should be styled in a similar manner to other pages.
- [ ] Page should be responsive and use Bulma

### PR DESCRIPTION
**Pull request by lyallstewart**
## What issue is this solving?

Closes #32 

### Description
Responsively styles the sharing code page.
image
image

## Any helpful knowledge/context for the reviewer?

- Any new dependencies to install? ❌ 
- Any special requirements to test? ❌ 

### Please make sure you've attempted to meet the following coding standards

- [X] Code has been tested and does not produce errors
- [X] Code is readable and formatted
- [X] There isn't any unnecessary commented-out code

### CODE REVIEW

File: sharing_code.html [hunk 1]
```diff
@@ -4,42 +4,80 @@
...
+                <p>For a larger chunk of code, wrap it in three backticks (`) with the name of the language:</p>
+                <img src="./assets/img/snippet.png">
+                <p>This has a number of benefits such as improved readability and syntax highlighting.</p>
+            </div>
+        </section>
+        <section class="container has-background-white mb-6 p-6">
+            <div>
+                <h2 class="is-size-2 p-0">Larger Code Files</h2>
+                <p>If you need to send a larger code snippet or a whole file, please avoid uploading it directly as a file (as file uploads are not permitted for security reasons) and instead use one of the options listed below:</p>
+                <section class="container">
```
Comment (aedwardg) [inline sharing_code.html:58]: I almost wonder if it looks better to have another `<hr>` here between the `<p>` tag and `<section>` tag so it doesn't look so cramped 🤔

File: sharing_code.html [hunk 2]
```diff
@@ -4,42 +4,80 @@
...
+                    <h3 class="is-size-3">Exporting Code from Codecademy</h3>
+                    <p>If you need to export code directly from the Codecademy workspace, you can click on the Tools header on the navbar and export your code to a gist, which you can then share a link to:</p>
+                    <img src="./assets/img/cc_sharing.png">
+                </section>
+                <hr>
+                <section class="container">
+                    <h3 class="is-size-3">Sharing off-platform code</h3>
```
Comment (aedwardg) [inline sharing_code.html:65]: I'm 50/50 on whether these headers should have numbers, like 
"1. Exporting Code from Codecademy" and 
"2. Sharing off-platform code".

I'm curious if you and @Victoria-DR think they look better with or without.

**Review by aedwardg — APPROVED** [review]
Great work as always @lyallstewart! This page looks so much nicer now!

Let's wait a couple days to see if anyone weighs in and if there are no requested changes by Wednesday (PST) I'll go ahead and merge this in.

**Comment by lyallstewart** [comment]
Thanks @aedwardg,
I think it looks better with both of those, I've updated them now 😄:
image

**Review by Victoria-DR — APPROVED** [review]
Looks great, thanks!

**Pull request merged**